apistructs/dicehub_release_rule.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apistructs
+
+import (
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+type BranchReleaseRuleModel struct {
```

Comment:
The model should not be defined under the `apistruct` package. Define the model in the `models` package of the functional module.

Reply:
Start a new pull request to fix it